.gitignore

```diff
@@ -9,3 +9,5 @@ docs/book
 release/*
 templates/cluster-template*.yaml
 infrastructure-*-linode/*
+*.yaml
+!metadata.yaml
```

Comment:
```suggestion
```
This will remove our manifests (e.g. `config/`, `templates/` from version control.

Reply:
This shouldn't recursively do that:
```
(⎈|kind-tilt:N/A) ~/cluster-api-provider-linode   client-timeout  vim .gitignore
(⎈|kind-tilt:N/A) ~/cluster-api-provider-linode   client-timeout  vim templates/addons/cilium/cilium.yaml
gst
(⎈|kind-tilt:N/A) ~/cluster-api-provider-linode   client-timeout ±  gst
On branch client-timeout
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   templates/addons/cilium/cilium.yaml
```